internal/agent/device/status/exporters_linux.go

```diff
@@ -0,0 +1,33 @@
+//go:build linux
+// +build linux
```

Comment:
Do we still need the old `// +build` style build constraints?

Reply:
ahh your right old habits I guess go 1.17+ does not